config/default.json

```diff
@@ -16,7 +16,8 @@
         "contrast": "contrastMode",
         "session": "blf-alpha-session",
         "csrf": "_csrf",
-        "abTest": "blf-ab-1-50"
+        "abTestHomepage": "blf-ab-1-50",
+        "abTestAwardsForAll": "blf-ab-afa"
```

Comment:
Worth putting an index on this too to reflect the chances of it being renamed/increased at some point?

Reply:
Not sure I follow?